nautobot/extras/migrations/0067_migrate_job_result_status.py

```diff
@@ -0,0 +1,31 @@
+from celery import states
+from django.db import migrations
+
+
+LEGACY_TO_NEW_JOB_RESULT_STATUS_MAPPING = [
+    ["pending", states.PENDING],
+    ["running", states.STARTED],
+    ["completed", states.SUCCESS],
+    ["errored", states.REVOKED],  # This is either REVOKED or RETRY. Not Sure
```

Comment:
?

Reply:
states.FAILURE, I'd think. I know it'll make things lossy on the reverse-migration front but I think that's OK.